frontend/src/metabase/admin/permissions/selectors/confirmations.ts

```diff
@@ -103,8 +107,14 @@ export function getControlledDatabaseWarningModal(
     getSchemasPermission(permissions, groupId, entityId, "data") !==
     "controlled"
   ) {
+    const [entityType, entityTypePlural] = isTableEntityId(entityId)
+      ? [t`table`, t`tables`]
+      : isSchemaEntityId(entityId)
+      ? [t`schema`, t`schemas`]
+      : [t`entity`, t`entities`];
```

Comment:
this is perfectly fine, but we also have a `pluralize` function in metabase/lib/formatting/strings.js`